src/cmd/src/datanode.rs

```diff
@@ -170,6 +170,7 @@ mod tests {
             ObjectStoreConfig::File { data_dir } => {
                 assert_eq!("/tmp/greptimedb/data/".to_string(), data_dir)
             }
+            _ => unreachable!(),
```

Comment:
Prefer enumerate all variants to avoid unexpected panic when new variant is added. 
```suggestion
            ObjectStoreConfig::S3 { .. } => unreachable!()
```